pkg/ansible/flags/flag.go

```diff
@@ -55,5 +58,12 @@ func AddTo(flagSet *pflag.FlagSet, helpTextPrefix ...string) *AnsibleOperatorFla
 			"Ansible verbosity. Overridden by environment variable."),
 			" "),
 	)
+	flagSet.StringVar(&aof.AnsibleRolesPath,
+		"ansible-roles-path",
+		"",
+		strings.Join(append(helpTextPrefix,
+			"(valid just for up local) - Ansible path roles ($ANSIBLE_ROLES_PATH). By default it will be /roles in the current directory"),
```

Comment:
This flag is valid for `up local`, but it _could_ also be passed as an argument in the operator's deployment YAML, right?.

```suggestion
			"Ansible path roles ($ANSIBLE_ROLES_PATH). By default it will be /roles in the current directory"),
```

Reply:
It shows true. Anyway, we created the issue: https://github.com/operator-framework/operator-sdk/issues/2369 to solve the doc scenario which was the motivation for this text.